### ISSUE TITLE
voice extension

### ISSUE THREAD

**Issue by Alexreid95** [fac19/week3-JICG#39]
1. Does not read list text, only checkbox so when I tab though the app, I don't know what the lists are for only whether its ticked or not  

2. When I click enter when adding a list item it says the list item is deleted
